### PR TITLE
[DW-314] Add blog post document type and UI

### PR DESCRIPTION
**Pull request by ngta1**
Includes a few updates for Cyber doc type value lists.

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    }
+
+    &__icon {
+        margin-right: 1.5rem;
+        width: 300px;
+        flex: none;
+
+        &__img {
+            width: 300px;
+            height: 200px;
```
Comment (DamZiobro) [inline repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss]: I fixing width and height good idea? I had similar problem in BlogHub and talked to Owain about it. He said we could fix width and crop on height. I implemented it that way using 'object-fit: cover' css. Is it worth to consider similar solution here?

File: repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        }
+
+        &__nhsimg {
+            width: 240px;
+            height: 140px;
```
Comment (DamZiobro) [inline repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss]: the same here

File: site/src/main/java/uk/nhs/digital/common/components/BlogComponent.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+
+public class BlogComponent extends BaseGaContentComponent {
+
+    @Override
+    public void doBeforeRender(final HstRequest request, final HstResponse response) throws HstComponentException {
+        super.doBeforeRender(request, response);
+
+        request.setAttribute("currentUrl", request.getRequestURL().toString());
```
Comment (DamZiobro) [inline site/src/main/java/uk/nhs/digital/common/components/BlogComponent.java]: Good to know we can do it this way ;)

File: repository-data/application/src/main/resources/hcm-config/configuration/update/EximImport.yaml [hunk 4]
```diff
@@ -2,12 +2,13 @@
 definitions:
   config:
     /hippo:configuration/hippo:update/hippo:registry/EximImport:
-      hipposys:batchsize: 10
+      hipposys:batchsize: 1
       hipposys:description: Exim import script modified by Valtech
       hipposys:dryrun: false
       hipposys:parameters: "{\r\n  \"fileInJson\": true,\r\n  \"sourceBaseFolderPath\"\
-        : \"file:${java.io.tmpdir}/exim-import/\"\r\n}"
+        : \"file:/tmp/migrator/exim-import/\"\r\n}"
```
Comment (DamZiobro) [inline repository-data/application/src/main/resources/hcm-config/configuration/update/EximImport.yaml]: Comment just to double check this change with fixed path is what we expect. Do we have the same fixed path for each test and production environment?

File: conf/query/lucene/indexing_configuration.xml [hunk 5]
```diff
@@ -179,6 +179,18 @@ https://wiki.apache.org/jackrabbit/IndexingConfiguration
...
+        <property>website:summary</property>
+        <property>website:shortsummary</property>
+        <property>website:seosummary</property>
+        <property>website:section</property>
+        <property>website:leadparagraph</property>
+        <property>website:backstory</property>
+
+        <property isRegexp="true" nodeScopeIndex="false">.*:.*</property>
+    </index-rule>
+
```
Comment (DamZiobro) [inline conf/query/lucene/indexing_configuration.xml]: what does it actually do?

File: repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss (hunk 1 above)
Comment (ngta1) [reply repository-data/webfiles/src/main/resources/site/scss/components/_blog.scss]: Good suggestion.  Will update.

File: repository-data/application/src/main/resources/hcm-config/configuration/update/EximImport.yaml (hunk 4 above)
Comment (ngta1) [reply repository-data/application/src/main/resources/hcm-config/configuration/update/EximImport.yaml]: This file not part of the PR, so will remove.

File: conf/query/lucene/indexing_configuration.xml (hunk 5 above)
Comment (ngta1) [reply conf/query/lucene/indexing_configuration.xml]: The indexing-configuration allows to define which doc type/fields are to be included in the search.  The last line excludes everything else (so only index the fields defined).

Comment (DamZiobro) [reply conf/query/lucene/indexing_configuration.xml]: (y)

**Pull request merged**